src/zjs_aio_k64f.c

```diff
@@ -0,0 +1,331 @@
+// Copyright (c) 2017, Intel Corporation.
+
```

Comment:
Shouldn't this work for all single-core boards, not just k64f? Let's just name this aio maybe and the other one a101 or ipm?

Reply:
i could rename them